### DISCUSSION TITLE
Flugplatzkarten

### DISCUSSION THREAD

**Discussion by Heinrich-Ffm** [Akaflieg-Freiburg/enroute#400] [Q&A]
Neben den VFR-Anflugkarten würde ich gern die Flugplatzkarten anzeigen. Ist das möglich?

**Comment by kebekus** [comment]
@Heinrich-Ffm Ja. Du kannst GeoTIFF-Dateien importieren (Randbedingung in der aktuellen Version: die Karten müssen genordet sein; die Randbedingung existiert in der nächsten Version nicht mehr). Die Karten werden dann unter "Anflugkarten" angezeigt, können natürlich aber auch Details zum Flugplatz zeigen.

In einer der nächsten Version wird der AIP-Browser DE in der Lage sein, Flugplatzkarten in die TripKits zu integrieren, die dann von Enroute verwendet werden können.
